README.md

```diff
@@ -80,22 +79,16 @@ systemctl restart containerd
 Or you can start nydus-snapshotter manually.
 
 ```bash
-# `nydusd-path` is the path to nydusd binary. If `nydusd` and `nydus-image` are installed, `--nydusd-path` and `--nydusimage-path`can be omitted.
+# `nydusd-path` is the path to nydusd binary. If `nydusd` and `nydus-image` are installed and can be searched from `PATH`, `--nydusd-path` and `--nydusimage-path`can be omitted.
```

Comment:
Why still `--nydusd-path`, `--nydusimage-path` here?

Reply:
Another PR https://github.com/containerd/nydus-snapshotter/pull/339/files is addressing this, which is clean up cli help message